### PR TITLE
1 new category and 7 api added

### PR DESCRIPTION
**Pull request by Ahmtdnz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,7 +7,6 @@ This is an attempt to categorise different APIs scoured from the web which make
 > *Items marked with ![Open Source](https://raw.githubusercontent.com/abhishekbanthia/Public-APIs/master/opensource.png "Open Source") are open-source
 > *Items marked with 💸 are trial based APIs
 
-
```
Comment (abhishekbanthia) [inline README.md]: Can you remove this whitespace deletion?

File: README.md [hunk 2]
```diff
@@ -175,6 +180,7 @@ APIs
 ### Image Moderation
 
 - [WebPurify](https://www.webpurify.com/image-moderation/) - Provides live image moderation by humans. Free for first 100 images. 2¢ per image after. 💸
+- [Dribble](http://developer.dribbble.com/) - Documentation and resources for developers building applications with the Dribbble API.
```
Comment (abhishekbanthia) [inline README.md]: Dribble is a great addition; but I don't believe it comes under the Image Moderation category.

File: README.md [hunk 3]
```diff
@@ -275,6 +281,9 @@ APIs
 ### News Services
 
 - [NewsAPI](https://newsapi.org/) - 70+ news sources and their headlines, one API.
+- [New York Times](http://developer.nytimes.com/) -  New York Times developer network and  API.
```
Comment (abhishekbanthia) [inline README.md]: Great additions. 👍

File: README.md [hunk 4]
```diff
@@ -193,7 +199,7 @@ APIs
 - [LinkedIn](https://developer.linkedin.com/docs/signin-with-linkedin) - Make it convenient for your users to sign in with their professional identity.
 - [Google Identity](https://developers.google.com/identity/) - Build an auth system that will grow with your business and reduce the friction of getting users into your app.
 - [Instagram](https://www.instagram.com/developer/authentication/) - Access to Instagram’s OAuth API.
-- [PayPal](https://developer.paypal.com/docs/integration/direct/identity/log-in-with-paypal/) - Enables your customers to sign in to your web site quickly and securely using their PayPal login credentials.
```
Comment (abhishekbanthia) [inline README.md]: Looks like an unnecessary deletion/addition.

File: README.md (hunk 2 above)
Comment (Ahmtdnz) [inline README.md]: Dribble category changed.

File: README.md (hunk 1 above)
Comment (Ahmtdnz) [inline README.md]: Whitespace removed.

**Comment by abhishekbanthia** [comment]
The Dribble API doesn't belong under the IOT category as well. Also, your fork has conflicts and needs to be updated with the base branch.

**Pull request closed without merging**